conf.js

```diff
@@ -9,9 +11,14 @@ exports.config = {
       },
       url: {
         urla: 'https://hotline.ua/'
-      }
+      },
+      register: {
+        email: 'jimbo'+Date.now()+'@gmail.com',
```

Comment:
should be `'jimbo+'+Date.now()+'@gmail.com'` to get all emails redirected to jimbo@gmail.com

Reply:
done